pkg/apis/pipeline/v1beta1/pipeline_types.go

```diff
@@ -90,6 +90,23 @@ type PipelineResult struct {
 	Value string `json:"value"`
 }
 
+type Metadata struct {
+	// +optional
+	Labels map[string]string `json:"labels,omitempty" protobuf:"bytes,1,rep,name=labels"`
+
+	// +optional
+	Annotations map[string]string `json:"annotations,omitempty" protobuf:"bytes,2,rep,name=annotations"`
+}
```

Comment:
@vdemeester @ImJasonH I have added a new type here, PTAL 🙇‍♀️

Reply:
nit: I don't think we need the `protobuf:` part :thinking:

Reply:
Done 😄